src/Misc/layoutbin/en-US/strings.json

```diff
@@ -222,12 +222,13 @@
     "",
     "Options:",
     "-----------------------------------------------------------------------------",
-    "--unattended     Unattended config. You will not be prompted.",
-    "                 All answers must be supplied to the command line.",
-    "--acceptteeeula  Accept TEE end user license agreement. (Only on Linux and OSX)",
-    "--replace        Replace the agent in a pool. If another agent is listening",
-    "                 by that name, or it will start failing with a conflict.",
-    "--runasservice   Configure the agent as service. (Only on Windows, need administrator permission to configure windows service)",
+    "--unattended       Unattended config. You will not be prompted.",
+    "                   All answers must be supplied to the command line.",
+    "--acceptteeeula    Accept TEE end user license agreement. (Only on Linux and OSX)",
+    "--replace          Replace the agent in a pool. If another agent is listening",
+    "                   by that name, or it will start failing with a conflict.",
+    "--runasservice     Configure the agent as service. (Only on Windows, need administrator permission to configure windows service)",
+    "--enableautologon  Enable autologon on the machine. (Only on Windows and when the agent is running as process)",
```

Comment:
Configure autologon and run agent on startup. (Only on Windows, need administrator permission to configure autologon)